interface/forms/clinical_notes/save.php

```diff
@@ -89,8 +89,9 @@
             // the note).
             $record['user'] = $_SESSION["authUser"];
         }
-//        $record['note_related_to'] = parse_note($record['description']);
-        $record['note_related_to'] = $record['description'];
+        // this is for related issues to the note
+        $record['note_related_to'] = parse_note($record['description']);
+        //$record['note_related_to'] = $record['description'];
```

Comment:
@adunsulag This is important. not sure why you removed.

Reply:
@sjpadgett  I'm really confused.  I remember doing work in here and I can't for the life of me remember why I would have commented this out as I do know its important.  My only thought is somehow I was in the middle of a refactor and forgot to save or got pulled into something else.  Sorry.

Reply:
no biggy, just wanted to ensure it wasn't breaking api...